buildtools/checkpackages/FSharp.Compiler.Service_notshipped.fsproj

```diff
@@ -0,0 +1,23 @@
+﻿<Project Sdk="Microsoft.NET.Sdk">
+
+  <Import Project="$(MSBuildProjectDirectory)\..\..\eng\Versions.props" />
+ 
+  <PropertyGroup>
+    <TargetFramework>net7.0</TargetFramework>
+    <DisableImplicitFSharpCoreReference>true</DisableImplicitFSharpCoreReference>
+    <CachePath>$(MSBuildProjectDirectory)\..\..\artifacts\tmp\$([System.Guid]::NewGuid())</CachePath>
```

Comment:
These few lines shouldn't need to be here; the auto-import of `Directory.Build.props` should handle it and reduce the duplication.

Reply:
I'm not a fan of putting project stuff in the Directory.Build.props and targets, it makes project files so hard to reason about.